lib/SDIR/Translate/translateToSDFG.cpp

```diff
@@ -0,0 +1,1597 @@
+  llvm::raw_string_ostream nameStream(name);
+  op->getResult(0).printAsOperand(nameStream, state);
+
+  jemit.startNamedObject(name);
+  jemit.printKVPair("type", "Array");
+
+  jemit.startNamedObject("attributes");
+  // TODO: Print the values you can derive
+  // jemit.printKVPair("allow_conflicts", "false", /*stringify=*/false);
+  // jemit.startNamedList("strides");
+  // jemit.endList(); // strides
+
+  // jemit.printKVPair("total_size", "4");
+  // jemit.startNamedList("offset");
+  // jemit.endList(); // offset
+
+  // jemit.printKVPair("may_alias", "false", /*stringify*/false);
+  // jemit.printKVPair("alignment", 0, /*stringify*/false);
+  jemit.printKVPair("dtype", "int32");
+  // jemit.startNamedList("shape");
+  // jemit.endList(); // shape
+
+  jemit.printKVPair("transient", "true", /*stringify=*/false);
+  jemit.printKVPair("storage", "Default");
+  jemit.printKVPair("lifetime", "Scope");
+  // jemit.startNamedObject("location");
+  // jemit.endObject(); // location
+
+  // jemit.printKVPair("debuginfo", "null", /*stringify=*/false);
+
+  jemit.endObject(); // attributes
+  jemit.endObject();
+  return success();
+}
+
+//===----------------------------------------------------------------------===//
+// GetAccessOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult translateGetAccessToSDFG(GetAccessOp &op, JsonEmitter &jemit) {
+  jemit.startObject();
+  jemit.printKVPair("type", "AccessNode");
+  jemit.printKVPair("label", op.getName());
+
+  jemit.startNamedObject("attributes");
+  jemit.printKVPair("access", "ReadWrite");
+  jemit.printKVPair("setzero", "false", /*stringify=*/false);
+  jemit.printKVPair("data", op.getName());
+  jemit.startNamedObject("in_connectors");
+  jemit.endObject(); // in_connectors
+  jemit.startNamedObject("out_connectors");
+  jemit.endObject(); // out_connectors
+  jemit.endObject(); // attributes
+
+  jemit.printKVPair("id", op.ID(), /*stringify=*/false);
+  jemit.printKVPair("scope_entry", "null", /*stringify=*/false);
+  jemit.printKVPair("scope_exit", "null", /*stringify=*/false);
+  jemit.endObject();
+  return success();
+}
+
+//===----------------------------------------------------------------------===//
+// LoadOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult translateLoadToSDFG(LoadOp &op, JsonEmitter &jemit) {
+  return success();
+}
+
+//===----------------------------------------------------------------------===//
+// StoreOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult translateStoreToSDFG(StoreOp &op, JsonEmitter &jemit) {
+  jemit.startObject();
+  jemit.printKVPair("type", "MultiConnectorEdge");
+
+  jemit.startNamedObject("attributes");
+  jemit.startNamedObject("data");
+  jemit.printKVPair("type", "Memlet");
+  jemit.startNamedObject("attributes");
+
+  jemit.printKVPair("volume", 1);
+  jemit.printKVPair("num_accesses", 1);
+
```

Comment:
use only `volume`, num_accesses is just a backwards compatibility field.

Also, this seems a bit hardcoded, especially since the range below can be larger than one.